packages/studio-base/src/components/SettingsTreeEditor/FieldEditor.tsx

```diff
@@ -337,18 +338,20 @@ function FieldEditorComponent({
   actionHandler,
   field,
   path,
+  style,
 }: {
   actionHandler: (action: SettingsTreeAction) => void;
   field: DeepReadonly<SettingsTreeField>;
   path: readonly string[];
+  style?: CSSProperties;
 }): JSX.Element {
   const theme = useTheme();
   const indent = Math.min(path.length, 4);
   const paddingLeft = theme.spacing(2 + 2 * Math.max(0, indent - 1));
 
   return (
     <>
-      <Stack direction="row" alignItems="center" style={{ paddingLeft }} fullHeight>
+      <Stack direction="row" alignItems="center" style={{ ...style, paddingLeft }} fullHeight>
```

Comment:
The Stack component has a paddingLeft prop that is tapped into the theme.spacing.

```suggestion
      <Stack direction="row" alignItems="center" style={style} paddingLeft={paddingLeft} fullHeight>
```